src/main/java/io/vertx/core/net/impl/NetClientImpl.java

```diff
@@ -226,6 +212,26 @@ private void connect(SocketAddress remoteAddress, String serverName, Promise<Net
   }
 
   public void connectInternal(ProxyOptions proxyOptions,
```

Comment:
Might be helpful to add JavaDoc to this method and the next?

Reply:
will do